docs/reference/snapshot-restore/register-repository.asciidoc

```diff
@@ -153,22 +153,44 @@ unit, for example: `1GB`, `10MB`, `5KB`, `500B`. Defaults to `null` (unlimited c
 
 The URL repository (`"type": "url"`) can be used as an alternative read-only way to access data created by the shared file
```

Comment:
This intro feels awkward & doesn't really explain why you would want to use a URL repository. Is the point that you can write snapshots to a shared file system repository, and then serve them up so they can be restored to nodes that don't have access to the shared filesystem?

Reply:
Added some more context with aa2b76ad0b9.

I don't think the primary issue is access but safety. A read-only shared file system could be accidentally switched to write, which could lead to corrupting snapshots if multiple clusters write to the same repository.